test/app-luatest/console_autorequire_test.lua

```diff
@@ -0,0 +1,171 @@
+local t = require('luatest')
+local it = require('test.interactive_tarantool')
+local cbuilder = require('test.config-luatest.cbuilder')
+local cluster = require('test.config-luatest.cluster')
+
+local g = t.group()
+
+local autorequire_list = {
+    'clock',
+    'compat',
+    'config',
+    'datetime',
+    'decimal',
+    'fiber',
+    'fio',
+    'fun',
+    'json',
+    'log',
+    'msgpack',
+    'popen',
+    'uuid',
+    'varbinary',
+    'yaml',
+}
+
+local function assert_no_autorequire(it)
+    it:roundtrip("require('strict').off()")
+
+    for _, m in ipairs(autorequire_list) do
+        it:roundtrip(('type(%s)'):format(m), 'nil')
```

Comment:
Why do you use `type(modulename)` here instead of just `modulename`? #9999 allows you to use the `nil` check in `roundtrip`, doesn't it?

Reply:
There is no idea behind it, I just implemented the positive and the negative checks similarly. Changed to `modulename`.